requirements-minimal.txt

```diff
@@ -22,5 +22,6 @@ PyYAML>=5.1
 requests
 task_processing[mesos_executor,k8s]
 Twisted>=19.7.0
+typing-extensions>=3.10.0.0
```

Comment:
we don't directly import typing_extensions, do we? requirements-minimal should only contain libraries that tron is directly importing, not transitive dependencies